pkg/shell/index.html

```diff
@@ -393,6 +393,19 @@ <h1 id="machine-error-heading"></h1>
 
     </div>
 
+    <div id="early-failure" class="blank-slate-pf" role="alertdialog" hidden>
+      <div class="blank-slate-pf-icon">
+        <i class="fa fa-exclamation-circle"></i>
+      </div>
+      <h1 translate>Connection failed</h1>
+      <p translate>There was an unexpected error while connecting to the machine.<br>
+        Messages related to the failure might be found in the journal: <code>journalctl -u cockpit</code>
+      </p>
+      <p translate>For Safari, you might want to import and trust the
+      certificate of the self-signing CA: <a download="ca.cer" href="/ca.cer">ca.cer</a>.
```

Comment:
Can you please check that the `<a>` is not part of the translatable string? This is very easy to get messed up. If it's included, please restrict the translatable part, through a nested `<span>`.

Reply:
I have removed that message.

Reply:
The message is back, and the link is not part of the text anymore.